docs/Anjay_integrations/Anjay_ST_integration/B-U585I-IOT02A-BG96.md

```diff
@@ -0,0 +1,94 @@
+# B-U585I-IOT02A/BG96
+
+Integrate your B-U585I-IOT02A Discovery kit board along with the default-provided Quectel BG96 modem.
+
+## Prerequisites
+
+- The STM32U585I-IOT02A/BG96 board with a USB cable.
+- Installed **STM32CubeIDE**.
+- Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
+- A user with access to the Coiote IoT Device Management platform.
+
+## Prepare binaries
```

Comment:
For what I gather here, we have two options to flash the board: using ready-made binaries or build the project in STM32CubeIDE. If so, I'd emphasise this by adding `### Option 1: Use already built binary` and `### Option 2: Start developement using samples` to the section titles.

Reply:
@mkrawczyk-avs I prefer to use note before the optional part. For me, it looks better on page.